### ISSUE TITLE
Github review comment should be "REQUEST_CHANGES" mode

### ISSUE THREAD

**Issue by rnitta** [prontolabs/pronto#319]
https://github.com/prontolabs/pronto/blob/899c6f270ed80cce1585abf1cf1b0006899021f3/lib/pronto/github.rb#L52

Now it is submitted with "COMMENT" mode.
it should be "REQUEST_CHANGES" (and this is also semantic).

Or, the mode should be selectable by user.

thoughts?

see also: 
https://octokit.github.io/octokit.rb/Octokit/Client/Reviews.html#create_pull_request_review-instance_method
https://developer.github.com/v3/pulls/reviews/#create-a-pull-request-review

**Comment by mikegee** [comment]
seems legit to me 👍

**Comment by mknapik** [comment]
I like the idea of submitting as Request Changes but making it default and only option seems to be pretty arbitrary.
I think it would be better to make the mode configurable so we don't impose any particular workflow on users.

**Comment by mmozuras** [comment]
I agree with @mknapik - "request changes" makes sense as the default and I think that we should be it configurable.

**Comment by ashkulz** [comment]
This has been implemented.
